src/linearmixedmodel.jl

```diff
@@ -644,8 +644,6 @@ function objective(m::LinearMixedModel{T}) where {T}
     isempty(wts) ? val : val - T(2.0) * sum(log, wts)
 end
 
-StatsBase.predict(m::LinearMixedModel) = fitted(m)
```

Comment:
Could this method and the corresponding method for `GeneralizedLinearMixedModel` be combined as
```
StatsBase.predict(m::MixedModel) = fitted(m)
```